CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 3.2.1: 2020-11-18
+
+Add support to configure number of agents to spawn with launchd on macOS by setting the --spwan flag.
```

Comment:
```suggestion
Fix: macOS can run `buildkite_agent_count` job-runners now.
```

Reply:
Oh, I see! Sorry, I misunderstood. In that case, yes, if you wouldn't mind swapping over to `buildkite_agent_spawn` throughout (as in, this PR, I'll do the other two platforms at some stage) and dropping `_count`, that would be very welcome.

Reply:
My code suggestions are based on thinking `buildkite_agent_count` existed and was active; please replace to `buildkite_agent_spawn` instead.

Reply:
Yep, my bad. This is basically all that the PR needed for this to work:

```diff
diff --git a/templates/buildkite-agent.cfg.j2 b/templates/buildkite-agent.cfg.j2
index e18622b..239e986 100644
--- a/templates/buildkite-agent.cfg.j2
+++ b/templates/buildkite-agent.cfg.j2
@@ -4,6 +4,9 @@ token="{{ buildkite_agent_token }}"
 # The name of the agent
 name="%hostname-%n"

+# Number of agents to spawn
+spawn="{{ buildkite_agent_count | default('1') }}"
+
```

Now I feel stupid, but I guess this is better. Will push these changes, if you still want to manage a plist file let me know!